spec/lib/administrate/search_query_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "spec_helper"
+require "administrate/search"
+
+describe Administrate::Search::Query do
```

Comment:
I think you can ignore it in the spec. It's a bit silly to restrict the size of your test!

The rule should be ignored for `spec/`, but I'll have a look at that.